refactor to allow objects and references in phpserialised data

ITJamie opened the pull request:
This allows Objects and References in phpserialised data.
It converts it to a custom string data type.

im happy to create this as an alternative phpserialize operator since its not exactly standard instead of extending the existing one

ITJamie committed:
refactor to allow objects and references in phpserialised data

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/gchq/CyberChef?pullRequest=2046) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/gchq/CyberChef?pullRequest=2046) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/gchq/CyberChef?pullRequest=2046) it.</sub>

ITJamie committed:
php deserializer fixing tests

as its now an object thats being converted to a json string the order of keys is sorted in js

ITJamie committed:
adding more tests + fixing again

ITJamie committed:
Update PHP.mjs

ITJamie committed:
Update PHP.mjs

ITJamie committed:
Update PHP.mjs

ITJamie committed:
Update PHP.mjs

ITJamie committed:
Update PHP.mjs

ITJamie committed:
refactor to have support for past output style

ITJamie committed:
adjust test

ITJamie committed:
linting fix

C85297 committed:
Merge branch 'master' into jamie-working2

ITJamie committed:
Merge branch 'master' into jamie-working2

ITJamie committed:
Merge branch 'master' into jamie-working2

ITJamie commented:
Is there anything else i need to do to get this reviewed?
